src/graphql/auth/auth.type.ts

```diff
@@ -1,14 +1,14 @@
-import { Field, InputType } from 'type-graphql';
+import { Field, ArgsType } from 'type-graphql';
 
-enum AuthType {
+export enum Provider {
   email = 'email',
   GOOGLE = 'google',
 }
 
-@InputType()
-class SignInInput {
+@ArgsType()
+export class SignInArgs {
   @Field()
```

Comment:
아래처럼 해야 에러 안날거야! 원시타입을 제외하고 우리가 직접 선언한것들은 type-graphql 에다가 직접 타입 선언해 줘야해
```suggestion
  @Field(type => Provider)
```

Reply:
아 우리가 직접 enum 선언한거 말하는거구나 ㅇㅋㅇㅋ